optuna/storages/rdb/storage.py

```diff
@@ -165,6 +165,8 @@ def get_study_id_from_name(self, study_name):
         session = self.scoped_session()
 
         study = models.StudyModel.find_or_raise_by_name(study_name, session)
+        # Terminate transaction explicitly to avoid connection timeout during query.
```

Comment:
I think `during transaction.` is more precise than `during query.`

Reply:
Sounds good! I fixed it (https://github.com/pfnet/optuna/pull/504/commits/ec8347f4ef9b90507175fd12fefa1c8a28742388).